plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -715,6 +717,11 @@ public ContainerCreated createContainer(CreateContainerParams createContainerPar
       LOG.info(e.getMessage());
       imageIdToWorkspaceId.remove(imageForDocker);
       openShiftDeploymentCleaner.cleanDeploymentResources(deploymentName);
+      if (! retryingAfterCleanup.get()) {
```

Comment:
@davidfestal could you please explain why you initially set `retryingAfterCleanup` = false, and than checks it as negative `!retryingAfterCleanup.get()`?  maybe I missing smth. but wouldn't it be more straight-forward to set it initially to true, and check without "!'

Reply:
Then I would have to name the TLS `canRetryOnce` :-) I can do it sure.

Reply:
approving, just thinking that probably better naming for `retryingAfterCleanup` can be done